### PR TITLE
Add Kovan WASM activation blocknumber

### LINKED ISSUE #7292: Tracking issue of wasm support in Kovan
- [ ] merge https://github.com/paritytech/parity/pull/7970
- [x] augment call to include gas limit #7356
- [x] refactor runtime to avoid allocation in contract memory
- [x] drop allocations from runtime 
- [x] audit functionality to reduce risk of further hardforks
- [x] ban floats
- [x] test the hardfork
- [x] deterministic stack limiter
- [x] decide block number 6,500,000

### PR DESCRIPTION
**Pull request by NikVolf**
closes #7292 

*** REQUIRES 7970 ***

### CODE REVIEW

File: ethcore/res/ethereum/kovan.json [hunk 1]
```diff
@@ -40,7 +40,8 @@
 		"eip140Transition": 5067000,
 		"eip211Transition": 5067000,
 		"eip214Transition": 5067000,
-		"eip658Transition": 5067000
+		"eip658Transition": 5067000,
+		"wasmActivationTransition": 6500000
```
Comment (tomusdrw) [inline ethcore/res/ethereum/kovan.json]: Should we fork on Friday? `6262000`?

Comment (5chdn) [reply ethcore/res/ethereum/kovan.json]: No, release will not be ready before Wed, we need at least 1 day for binaries and 1 day for tests

Comment (tomusdrw) [reply ethcore/res/ethereum/kovan.json]: Ok, let's fork on next Wed then?

Comment (andresilva) [reply ethcore/res/ethereum/kovan.json]: #6350000 ?

Comment (NikVolf) [reply ethcore/res/ethereum/kovan.json]: Yeah, 6500000 is safer
we add might have some integration tests (in dedicated repos) meanwhile

**Pull request merged**